main.js

```diff
@@ -0,0 +1,12 @@
+var day = document.querySelector('.week');
+var schedule = document.querySelector('.event-viewer');
+
+day.addEventListener('click', dayOfWeek);
+
+function dayOfWeek() {
```

Comment:
If you are going to use the event object specifiy the parameter so you are not checking the global event object.